index.html

```diff
@@ -13,47 +13,52 @@
 </head>
 
 <body>
-    <div class="headline-container">
-        <!-- Header and navbar -->
+    <div class="hero-container">
         <header class="header">
-            <nav>
-                <img src="./assets/Toolbar - Dark - Compact.png" alt="">
-            </nav>
-        </header>
-        <!-- Hero headline -->
-        <section class="headline">
-            <h1 class="headline__title"><span class="headline__title--black">Hey there. <br>I'm Cuau. <br><span
-                        class="headline__title--green">I'm a
-                        software developer</span></span></h1>
-            <p class="headline__text"> I can help you build a product , feature or website. Look through some of my work
-                and experience! If you like what you see and have a project you need coded,
-                don’t hesitate to contact me. </p>
-            <div class="social-icons">
+            <img src="./assets/Toolbar - Dark - Compact.png" alt="">
+            <nav class="header__nav">
                 <ul>
-                    <li><a href="https://twitter.com/cuau_cabrera"><img src="./assets/twitter.png"
-                                alt="This is a Twitter icon. Click on it to visit my Twitter profile"
-                                class="headline__icons--twitter"></a></li>
-                    <li><a href="https://linkedin.com/in/cuaucabrera"><img src="./assets/Linkedin icon.png"
-                                alt="This is a Linkedin icon. Click on it to visit my Linkedin profile"
-                                class="headline__icons--linkedin"></a></li>
-                    <li><a href="https://medium.com"><img src="./assets/medium.png"
-                                alt="This is a medium icon. Click on it to visit my medium profile"
-                                class="headline__icons--medium"></a></li>
-                    <li><a href="https://github.com/cuauh-cabrera"><img src="./assets/github.png"
-                                alt="This is a github icon. Click on it to visit my github profile"></a></li>
```

Comment:
- [ ] Your project has some issues with the work/project sections as mentioned by the previous reviewer, the image below best describes what the issues are 👇🏾 

The issue is from many angles, please try to fix it

Mobile
![image](https://github.com/cuauh-cabrera/my_portfolio/assets/84156714/70392c9e-5189-41c8-ade9-bd2069541ca3)

Desktop
![image](https://github.com/cuauh-cabrera/my_portfolio/assets/84156714/8332f2ce-4531-40a4-8856-f4bc6601630e)


Please fix it.

Reply:
@bobb-Rob Thanks for your feedback. As per the images attached, I can’t see the issue with the project's card. In the images below is not possible to see the problem in the mobile version, please notice that the images come from three different web browsers (Firefox, Chrome, and Safari). 
![Captura de Pantalla 2023-06-09 a la(s) 12 49 31](https://github.com/cuauh-cabrera/my_portfolio/assets/114158910/de4df6a1-fd48-4814-976e-06c93b8ff247)
![Captura de Pantalla 2023-06-09 a la(s) 12 54 44](https://github.com/cuauh-cabrera/my_portfolio/assets/114158910/3b41a398-86c1-467e-b608-0a89bca6b182)
![Captura de Pantalla 2023-06-09 a la(s) 12 56 07](https://github.com/cuauh-cabrera/my_portfolio/assets/114158910/371ed6bc-b5c5-4905-8095-06bb84e07f53)